modelkit/assets/manager.py

```diff
@@ -218,7 +224,7 @@ def fetch_asset(
         self,
         spec: Union[AssetSpec, str],
         return_info=False,
-        force_download: Optional[bool] = None,
+        force_download: bool = None,
```

Comment:
Too bad for the Optional

Reply:
the one good Optional. I really thought they were no longer  necessary, although it is the other way around